libs/designsystem/form-field/src/form-field.component.ts

```diff
@@ -190,5 +193,57 @@ export class FormFieldComponent
     this.affixElements.forEach((affix) => {
       this.resizeObserverService.unobserve(affix.el);
     });
+
+    this.nestedInteractiveErrorSubscription.unsubscribe();
+  }
+
+  private initializeNestedInteractiveElement() {
+    this.nestedInteractiveElement =
+      this.input?.nativeElement ||
+      this.textarea?.nativeElement ||
+      this.radioGroupElement?.nativeElement.querySelector('ion-radio-group');
+  }
+
+  private setNestedInteractiveElementAttributes() {
+    if (!this.nestedInteractiveElement) return;
+
+    if (this.message) {
+      this.renderer.setAttribute(
+        this.nestedInteractiveElement,
+        'aria-describedby',
+        this._messageId
+      );
+
+      this.renderer.setAttribute(
+        this.nestedInteractiveElement,
+        'aria-errormessage',
+        this._errorMessageId
+      );
+    }
+
+    if (this.label && this.radioGroupElement) {
+      this.renderer.setAttribute(this.nestedInteractiveElement, 'aria-labelledby', this._labelId);
+    }
+  }
+
+  private subscribeToNestedInteractiveError() {
+    const nestedInteractiveComponent =
+      this.inputComponent || this.textareaComponent || this.radioGroupComponent;
+
+    this._nestedInteractiveHasError = nestedInteractiveComponent?.hasError;
+
+    this.nestedInteractiveErrorSubscription = nestedInteractiveComponent?.hasErrorChange.subscribe(
```

Comment:
Add question mark if hasErrorChange is not defined in nestedInteractiveComponent
```suggestion
    this.nestedInteractiveErrorSubscription = nestedInteractiveComponent?.hasErrorChange?.subscribe(
```

Reply:
The nestedInteractiveComponent is strongly typed due to this line above:
```
const nestedInteractiveComponent =
      this.inputComponent || this.textareaComponent || this.radioGroupComponent;
```